src/transport/helpers/check-if-complete.js

```diff
@@ -11,35 +11,31 @@ function checkIfComplete(transporter, resultAggregator, onCompleteCallback) {
   var internalState = transporter.internalState;
 
   if (resultAggregator.web3Transports.some(isUndefined)) return;
-  if (resultAggregator.syncTransports.some(isUndefined)) return;
   if (resultAggregator.httpTransports.some(isUndefined)) return;
   if (resultAggregator.wsTransports.some(isUndefined)) return;
   if (resultAggregator.ipcTransports.some(isUndefined)) return;
 
-  if (resultAggregator.syncTransports.every(isNull)
-    && resultAggregator.web3Transports.every(isNull)
+  if (resultAggregator.web3Transports.every(isNull)
     && resultAggregator.httpTransports.every(isNull)
     && resultAggregator.wsTransports.every(isNull)
     && resultAggregator.ipcTransports.every(isNull)) {
     return onCompleteCallback(new Error("Unable to connect to an Ethereum node via any tranpsort (Web3, HTTP, WS, IPC)."), null);
```

Comment:
Typo: "tranpsort". Looks like there are some errors in other files containing this as well.